commandline/src/main/java/tech/pegasys/web3signer/commandline/subcommands/Eth2SubCommand.java

```diff
@@ -125,6 +133,28 @@ public Runner createRunner() {
         isKeyManagerApiEnabled);
   }
 
+  private void logNetworkSpecInformation() {
+    LOG.info("Network: {}", network);
+    final ForkSchedule forkSchedule = eth2Spec.getForkSchedule();
+    final Map<SpecMilestone, UInt64> milestoneSlotMap =
+        forkSchedule
+            .streamMilestoneBoundarySlots()
+            .collect(Collectors.toMap(TekuPair::getLeft, TekuPair::getRight));
+    forkSchedule
+        .getActiveMilestones()
+        .forEach(
+            m -> {
+              final String specName = m.getSpecMilestone().name();
+              final UInt64 forkEpoch = m.getFork().getEpoch();
+              final UInt64 boundarySlot = milestoneSlotMap.get(m.getSpecMilestone());
+              LOG.info(
+                  "Spec Name: {}, Fork Epoch: {}, Boundary Slot: {}",
```

Comment:
Can't see "Boundary" in the teku docs but the consensus specs do mention "fork boundary". 
It's not a blocker, but maybe "First slot" would be more intuitive for readers of this log?

Reply:
👍🏼